### ISSUE TITLE
Uncaught RangeError: Invalid array buffer length

### ISSUE THREAD

**Issue by lightmar** [nusofthq/Recordmp3js#8]
Hi,

Thank you for this great plugin.
Unfortunately I have a problem which I can't figure out.
On some computers it works great, however on others when the recorded wav is passed to the libmp3lame, to I get the following error:
Uncaught RangeError: Invalid array buffer length

I have discovered that the error occurs only when a user records microphone with a high sample rate of 192000. When I try it on my mac with 44000 input, it works.

How can it be fixed? Very important issue...

Thanks

**Comment by michelboudali** [comment]
@lightmar did you find how to reduce the sample rate?

**Comment by z-shan** [comment]
@lightmar @michelboudali yeah it doesnt work for me too on windows machine when i try to record in 32kbps bitrate. 64kbps works on windows and 32 works on mac..
any fix for this. I would like to record with 32kbps to keep the size low.
